lib/webmock/request_signature.rb

```diff
@@ -44,6 +44,10 @@ def json_headers?
       !!(headers&.fetch('Content-Type', nil)&.start_with?('application/json'))
     end
 
+    def parsed_body
```

Comment:
@johngallagher  Thank you for this feature. I understand the intention behind this method. I have some concerns though. The current implementation assumes JSON, but request bodies can come in various formats (e.g., form-url-encoded, XML). This method would raise errors for non-JSON payloads. Additionally, while symbolized keys can be useful, it's often a matter of user preference rather than a universal expectation.

To address these issues, we could either expand `parsed_body` to support multiple formats, or allow users to apply their own decorators for custom parsing logic. Given these considerations, I suggest treating this as a separate feature. This would allow us to design a more flexible solution that caters to various use cases and preferences.

What are your thoughts on this approach?

Reply:
Yes, let's treat this as a separate feature. A massive overreach on my part! I'll rip it out.

Reply:
Ripped out.